content/account/organizations.textile

```diff
@@ -0,0 +1,178 @@
+---
+title: Organizations
+meta_description: "Manage Ably organizations, provision users, configure SSO with SCIM, and handle account roles and permissions."
+meta_keywords: "Ably organizations, user provisioning, SSO, SCIM, access management, Ably dashboard, account roles, permissions"
+---
+
+Use organizations to manage multiple Ably accounts by centralizing user access and permissions under a single organizational structure. Organizations streamline user provisioning through a Single Sign-On (SSO) configuration and utilize System for Cross-domain Identity Management (SCIM) for group-based access control.
+
+Organizations enable you to assign roles and permissions across different accounts, such as production, staging, and development environments.
+
+<aside data-type='note'>
+<p>An "Enterprise":/pricing/enterprise account is required to use organizations. "Contact us":https://ably.com/contact to enable organizations for your account.</p>
+</aside>
+
+h2(#primary). Primary account
+
+The primary account is an organization's main account and holds the following privileges:
+
+* The highest level of access to the organization.
+* Ownership of all all accounts within the organization.
+* The default account used for provisioning.
+
+h2(#accounts). Create accounts in an organization
+
+Create "accounts":https://ably.com/accounts/any in an organization:
+
+* Open the *Account* navigation dropdown.
+** Click *Organization Accounts*.
+** Click *New account*.
+** Add an account name and *Create account*.
+
+h3(#isolate). Isolate environments
+
+Use separate accounts within an organization to create isolated environments, such as production, staging, and development. Assign each environment a "package":/pricing#packages that meets its specific needs. For example, production may need high capacity with an Enterprise package, staging might use a Standard package, and development a Free package.
+
+h2(#manage). Manage user account access
+
+"Managing user":/account/users account access allows control of both organizational management permissions and access to specific accounts within your organization.
```

Comment:
I'm not quite sure what this means. Is access to each account not handled within the account itself?